test/support/helpers/scheduled-notification.helper.js

```diff
@@ -37,7 +37,8 @@ function add (data = {}) {
  */
 function defaults (data = {}) {
   const defaults = {
-    eventId: generateUUID()
+    eventId: generateUUID(),
+    notifyStatus: 'delivered'
```

Comment:
If we've added a new default that should mean we need to document it in the JSDoc comments for the helper.